packages/react-components/react-virtualizer/src/components/VirtualizerScrollView/useVirtualizerScrollViewStyles.ts

```diff
@@ -0,0 +1,68 @@
+import { VirtualizerScrollViewState } from './VirtualizerScrollView.types';
+import type { SlotClassNames } from '@fluentui/react-utilities';
+import { VirtualizerScrollViewSlots } from './VirtualizerScrollView.types';
+import { useVirtualizerStyles_unstable, virtualizerClassNames } from '../Virtualizer/useVirtualizerStyles';
+import { makeStyles, mergeClasses } from '@griffel/react';
+
+const virtualizerScrollViewClassName = 'fui-Virtualizer-Scroll-View';
+
+export const virtualizerScrollViewClassNames: SlotClassNames<VirtualizerScrollViewSlots> = {
+  ...virtualizerClassNames,
+  container: `${virtualizerScrollViewClassName}__container`,
+};
+
+const useStyles = makeStyles({
+  base: {
+    display: 'flex',
+    width: '100%',
+    height: '100%',
+    overflowAnchor: 'none',
+  },
+  vertical: {
+    flexDirection: 'column',
+    overflowAnchor: 'none',
+    overflowY: 'auto',
+  },
+  horizontal: {
+    flexDirection: 'row',
+    overflowX: 'auto',
+  },
+  verticalReversed: {
+    flexDirection: 'column-reverse',
```

Comment:
We should not use flex-direction for this, if virtualized elements are focusable, that focus will still follow DOM order. That's the same for screenreader navigation. This would cause a11y bugs

Reply:
I'll have to think about this one a bit, I can see viable use-cases where a user would want the items order modified in the DOM (like a sort) - but wondering how it ties in with A11Y for things like teams where text scrolls from the bottom, I would assume the first node in DOM being at the bottom of the list is what accessibility would be expecting when we have a teams chat-box scenario, as well as that CSS providing out of the box optimizations like beginning the scroll view at the bottom / initializing at index 0 - I will need to do more research here on alternatives.

Reply:
> I would assume the first node in DOM being at the bottom of the list is what accessibility would be expecting when we have a teams chat-box scenario

Not at all, Teams sticks to DOM order. If something CSS based like flex order would be used then there would be bugs in most keyboard navigation scenarios where the `Up` arrow would move focus downwards and vice versa. I'll create an issue to not block this PR, but it should be addressed

Reply:
Tracking here https://github.com/microsoft/fluentui/issues/27285

Reply:
Agreed - spoke with Sarah and she helped fill me in on A11Y further, we will definitly resolve this via 'expected' DOM order and have to optimize for initializing at the end of scrollview ourselves for reversed lists. (As well as interfacing the abstraction to not confuse Virtualizer dev users).